### ISSUE TITLE
Problem parsing pose values

### ISSUE THREAD

**Issue by VictorLamoine** [moveit/moveit_setup_assistant#107]
When adding a custom robot pose with joint values different from zero, the values are saved with the locale separator (`,` in my case). This leads to an error when trying the `demo.launch` because the value cannot be parsed:

``` bash
[ERROR] [1442563400.767605554]: Unable to parse joint value '-1,5707'
```

This can be easily fixed by replacing the `,` with a `.` in the generated SRDF.

Something like that should fix the issue:

```
doublespinbox->setLocale(QLocale(QLocale::English));
```

**Comment by gavanderhoorn** [comment]
Duplicate of #51?

**Comment by VictorLamoine** [comment]
Yes it is a duplicate of that issue.

**Comment by rhaschke** [comment]
Fixed with #123.
